server/webhook_parser.go

```diff
@@ -439,7 +441,8 @@ func mergeWebhookEvents(events []*webhook) Webhook {
 		if event.fieldInfo.name == descriptionField || strings.HasPrefix(event.fieldInfo.from, strike) {
 			strike = ""
 		}
-		msg := "**" + strings.Title(event.fieldInfo.name) + ":** " + strike +
+		// Use the english language for now. Using the server's local might be better.
+		msg := "**" + cases.Title(language.English, cases.NoLower).String(event.fieldInfo.name) + ":** " + strike +
```

Comment:
I'm thinking `event.fieldInfo.name` is probably english so it makes sense to use that here. It could potentially be a translated value from GitHub. Either way, as long as this doesn't break anything `strings.Title` was doing, I think it's fine.